plugin/src/main/kotlin/org/gradle/dependencygraph/extractor/DependencyExtractor.kt

```diff
@@ -144,8 +145,9 @@ abstract class DependencyExtractor :
         }
         val projectIdentityPath = (rootComponent.id as? DefaultProjectComponentIdentifier)?.identityPath?.path
 
-        // TODO: At this point, any resolution not bound to a particular project will be assigned to the root "build :"
+        // At this point, any resolution not bound to a particular project will be assigned to the root "build :"
         // This is because `details.buildPath` is always ':', which isn't correct in a composite build.
+        // This is inconsequential for GitHub Dependency Graph, since all dependencies are mapped to a single manifest.
```

Comment:
Is there a plan to change this in the future? Or a discussion/issue capturing this? I think we discussed the rationale behind this before, but I forget.

Reply:
Well, this could be consequential for other mapping strategies. 
It could impact other output formats (like SBOM). But there's no current plan or rationale to change the default mapping for GitHub Dependency Graph.

I thought about adding an issue but it's pretty obscure: if you use a different mapping and you have included builds, then the plugin dependencies of those included builds may be attributed to the top-level build.